code/modules/mob/living/simple_animal/hostile/megafauna/hivetechboss.dm

```diff
@@ -11,8 +11,8 @@
 	pixel_x = -16
 	ranged = TRUE
 
-	health = 2250
-	maxHealth = 2250 //Only way for it to show up right now is via adminbus OR Champion call (which gives it 150hp). For comparison Kaiser has 2000hp
+	health = 1650
+	maxHealth = 1650 //Only way for it to show up right now is via adminbus OR Champion call (which gives it 150hp). For comparison Kaiser has 2000hp
```

Comment:
600 hp nerf is a lot, I wouldn't put it lower than the Kaiser.

Reply:
aight, will change it so it's the same as kaiser